### PR TITLE
[Documentation] manual Zeta openflow rule documentation for Issue 148

### PR DESCRIPTION
**Pull request by Zqy11**
This document records the configuration steps of the first package upload and group table.
I'm writing documents with Markdown format for the time being, as I'm not familiar with the AsciiDoc format.
I will change to AsciiDoc format later.

### CODE REVIEW

**Comment by Zqy11** [comment]
I have update the manual construction document, there are three major updates:
(1) Modify the network segment of vlan_id=200;
(2) Add configuration of direct path;
(3) The ARP rule is modified and use the default forwarding function (actions=NORMAL) of gateway for the time being.

File: docs/manual_construction.md [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+Use ovs-docker to add a nic for the container and hang it to the br-int.
+Set different vlan_id (100 and 200).**
+```
+root@computer7:/# docker run -itd --name docker71 --net none ubuntu /bin/bash
+root@computer7:/# ovs-docker add-port br-int eth0 docker71 --ipaddress=192.168.1.71/24 --mtu=1450
+root@computer7:/# ovs-vsctl show
+root@computer7:/# ovs-vsctl set port 3cf61d844bb54_l tag=100
+
+root@computer7:/# docker run -itd --name docker72 --net none ubuntu /bin/bash
+root@computer7:/# ovs-docker add-port br-int eth0 docker72 --ipaddress=192.168.2.72/24 --mtu=1450
```
Comment (er1cthe0ne) [inline docs/manual_construction.md:90]: I am curious on why you need to set the MTU to 1450? Below is the command I used in the test code aca_test_ovs_l3.cpp: 

  cmd_string = "ovs-docker add-port br-int eth0 con3 --ipaddress=" + vip_address_3 +
               "/24 --gateway=" + subnet1_gw_ip + " --macaddress=" + vmac_address_3;

You may want to set the gateway address for the later usage (L3 routing).

**Comment by Zqy11** [comment]
Because the host’s nic MTU is also 1500. We use VXLAN tunnel to connect two hosts. The &nbsp;VXLAN will add 50 bytes on Original L3 Packet. When &nbsp;the VM/container’s&nbsp;nic MTU set 1500. The total VXLAN packet will be up to 1550.
So the 1550 bytes VXLAN packet will be&nbsp;fragment by the host’s nic.&nbsp;
However, the existing VTEP equipment generally requires that VXLAN packet cannot be&nbsp;fragmented&nbsp;when unpacking VXLAN packet, otherwise it cannot be unpacked correctly and will be dropped.&nbsp;This requires a minimum MTU of 1550 bytes for all network devices between VTEPs.
On Monday when we test connectivity, we observed that the&nbsp;ping packet is OK(because ping packet is short). However, when we use iperf tool to send udp packets, the&nbsp;opposite docker cannot receive udp packets. If we set the udp packet length shorter, it will work. We spent a few hours learning related mechanisms.
We observed that OpenStack also set the VM’s nic MTU 1450.
So we don’t change host’s nic MTU, we set the&nbsp;container’s nic MTU 1450.






------------------ Original ------------------

**Comment by er1cthe0ne** [comment]
understood. Try to see if you can set the host nic MTU to 9000 (jumbo frame size) and make sure the hardware network switch can support it. In the datacenter environment because of encap (eg. vxlan), it is likely to set MTU 9000 or even higher.

**Comment by er1cthe0ne** [comment]
The next step for this is to merge this information into the Zeta design doc started by @HuaqingTu . I have merged that change into master for you to proceed.

**Review by er1cthe0ne — APPROVED** [review]
LGTM

**Comment by er1cthe0ne** [comment]
I like the detail and format of this document. I am going to merge it in also.

**Pull request merged**